### ISSUE TITLE
A common confusion is that people use the "crosswalk" label to indicate where there *should* be crosswalks

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2976]
I'm seeing this alot both in Oradell and other places (like La Piedad).

Perhaps if we want to support this behavior, we add in a tag that says "I think a crosswalk should exist here" or something like that? I realize this is a bit inconsistent with our "curb ramp" vs. "missing curb ramp" labels but maybe scales better...

![image](https://user-images.githubusercontent.com/1621749/181595770-46487cce-acd0-40bf-bd09-62c81b9c3d11.png)

**Comment by misaugstad** [comment]
The tag could really just be "missing crosswalk". I agree that I'd rather just have people not label the missing crosswalks, but they ultimately are going to do that, so we should have a tag for it.

**Comment by jonfroehlich** [comment]
Yah, I think having a tag here like "missing crosswalks" could be an opportunity to see how well something like this works...

**Comment by misaugstad** [comment]
@Z-Setiadi there should be quite a few merged PRs I've done where I just added a tag that you can look at and follow in order to make this change! The only difference is that I'd like for you to find good pictures for the tooltip that we show when hovering over a tag!

I added some of those tags quickly and didn't include the tooltip image. @hawksFTW is working on adding them for all the ones I skipped during that crunch time, but I'd like you to find a couple possible images and share them here so that we can pick the best one!

Note that the images are going to be in a relatively small tooltip, so it needs to be pretty clear at a glance that a crosswalk is needed in that situation.

**Comment by jonfroehlich** [comment]
I'd like to get this done for our next release even without an example image. This continues to be an issue and the longer we wait, the more data we collect with 'crosswalk' labels being used to label missing crosswalks.

![image](https://user-images.githubusercontent.com/1621749/186909404-094ded73-4115-4b29-ad5d-0bb1fdca5000.png)

**Comment by jonfroehlich** [comment]
This is also coming up in Auckland now... we need to figure out a strategy to address this. Basically, what Mikey told Louise (from Auckland) is that the lack of a label indicates that a crosswalk is missing. But it seems like folks are still compelled to try and mark areas where they think crosswalks should exist...

![image](https://user-images.githubusercontent.com/1621749/224061259-36783c86-9261-41a7-b9ad-6f082f09b085.png)

**Comment by jonfroehlich** [comment]
This continues to be an issue. Our users seem to really want to indicate where there _should_ be crosswalks. I don't know the best way to deal with this. :(

![image](https://user-images.githubusercontent.com/1621749/227722589-880c8769-4df5-486e-972c-08ab3a261dda.png)

It does make me think that maybe the language could be "Curb Ramp Needed Here" vs. "Missing Curb Ramp" and "Crosswalk Needed Here" vs. "Missing Crosswalk"—if we want to pursue some sort of redesign update to try and address this.

**Comment by misaugstad** [comment]
Yeah I like that! And then the first thing you do is say whether or not the curb ramp / crosswalk is there! We could of course continue to have convenience things like continuing to have keyboard shortcuts 'M' for missing curb ramp and 'C' for curb ramps, and it just auto-fills whether or not it exists. We could even set the color/design of the icons based on whether or not they say that it exists, it just comes from a single label type in the ribbon menu at the top!

**Comment by jonfroehlich** [comment]
Right. Exactly! Would be fun to play with ideas here.

Also, towards "crosswalk needed." Is it required that all intersections with curb ramps have painted crosswalks—even in residential neighborhoods? I could see folks using this label alot but perhaps not in places where crosswalks are required. Hmm.

**Comment by misaugstad** [comment]
We can always reopen later, but I think that we've settled on just doing #3519 for now (renaming to "Marked Crosswalk"). Keeps the number label types from ballooning, is easy-ish to implement, and adds clarity to prevent mislabeling.
